DHI build harness: skip TeX manual doc targets in offline groff builds.
texi2dvi cannot run in the hermetic builder; prebuilt doc/groff.{dvi,pdf,info}
from the orig tarball are copied into the build tree by dhi-skip-doc-rebuild-rules.patch.
Upstream source: https://www.gnu.org/software/groff/

diff --git a/doc/doc.am b/doc/doc.am
--- a/doc/doc.am
+++ b/doc/doc.am
@@ -514,8 +514,7 @@
 GROFF_PDF = doc/groff.pdf
 endif
 
-all: doc/groff.info doc/groff.txt doc/groff.html $(GROFF_DVI) \
-  $(GROFF_PDF)
+all:
 
 # Distribute the manual in source form as well.
 EXTRA_DIST += doc/groff.texi doc/fdl.texi
